pkg/virtctl/vmexport/vmexport.go

```diff
@@ -166,8 +192,13 @@ func usage() string {
 	{{ProgramName}} vmexport download vm1-export --volume=volume1 --output=disk.img.gz
   
 	# Create a VirtualMachineExport and download the requested volume from it
-	{{ProgramName}} vmexport download vm1-export --vm=vm1 --volume=volume1 --output=disk.img.gz`
+	{{ProgramName}} vmexport download vm1-export --vm=vm1 --volume=volume1 --output=disk.img.gz
 
+	# Create a VirtualMachineExport and get the VirtualMachine manifest in Yaml format
+	{{ProgramName}} vmexport manifest vm1-export --vm=vm1
+
+	# Get the VirtualMachine manifest in Yaml format from an existing VirtualMachineExport including CDI header secret
+	{{ProgramName}} vmexport manifest existing-export --include-secret`
```

Comment:
What do you think about adding a `--manifest` param to `download` instead of a dedicated `manifest` command?

Reply:
That is an interesting thought, I can easily switch the handling function if the flag is set. There are many common flags between the two. Let me think about it.

Reply:
Done actually.